# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /home/elink/Diabetech/Development/src/healthimo.com/healthimo/trunk/public_html/includes
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: file.inc
--- file.inc Base (BASE)
+++ file.inc Locally Modified (Based On LOCAL)
@@ -908,7 +908,7 @@
           // Give priority to files in this folder by merging them in after any subdirectory files.
           $files = array_merge(file_scan_directory("$dir/$file", $mask, $nomask, $callback, $recurse, $key, $min_depth, $depth + 1), $files);
         }
-        elseif ($depth >= $min_depth && ereg($mask, $file)) {
+        elseif ($depth >= $min_depth && @ereg($mask, $file)) {
           // Always use this match over anything already set in $files with the same $$key.
           $filename = "$dir/$file";
           $basename = basename($file);
